### PR TITLE
4532 Init Tab Home

### PR DESCRIPTION
**Pull request by TungPhanThanh**
Creat Fragment home, Adapter.
Load genre title, genre image to fragment home.

### CODE REVIEW

**Comment by TungPhanThanh** [comment]
ready

File: app/src/main/java/com/example/themo/musicmarvelous/ui/main/home/GenreAdapter.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        View view = LayoutInflater.from(parent.getContext())
+                .inflate(R.layout.item_genre, parent, false);
+        return new ViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull GenreAdapter.ViewHolder viewHolder, int i) {
+        Genre genre = mGenres.get(i);
+        viewHolder.bindView(genre);
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/themo/musicmarvelous/ui/main/home/GenreAdapter.java]: remove bl

File: app/src/main/java/com/example/themo/musicmarvelous/ui/main/home/GenreAdapter.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    @Override
+    public int getItemCount() {
+        return mGenres != null ? mGenres.size() : 0;
+    }
+
+    public class ViewHolder extends RecyclerView.ViewHolder {
+        TextView mTextViewGenre;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/themo/musicmarvelous/ui/main/home/GenreAdapter.java]: 49 50 Accesmodifier

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by TungPhanThanh** [comment]
ready

**Pull request merged**